src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.proj

```diff
@@ -1,11 +1,13 @@
 <Project Sdk="Microsoft.Build.NoTargets">
+
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IsPackable>true</IsPackable>
     <!-- Reference the outputs for the dependency nodes calculation. -->
     <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
     <!-- This is a meta package and doesn't contain any libs. -->
     <NoWarn>$(NoWarn);NU5128</NoWarn>
+    <PackageReadmeFilePath>$(MSBuildThisFileDirectory)../src/PACKAGE.md</PackageReadmeFilePath>
```

Comment:
Do you mind explaining why this is different? I see it's from a pkg folder, which is rare for a library to have.

Reply:
System.IO.Ports consists of:
1. A NuGet package for the lib -> https://www.nuget.org/packages/System.IO.Ports/9.0.0-preview.6.24327.7
2. A NuGet package for the native meta package -> https://www.nuget.org/packages/runtime.native.System.IO.Ports/
3. A NuGet package for the native RID specific package -> https://www.nuget.org/packages/runtime.linux-arm.runtime.native.System.IO.Ports/

The native packages should just re-use the package README from the lib package.